Sources/SwiftExtensions/AsyncQueue.swift

```diff
@@ -28,28 +28,31 @@ extension Task: AnyTask {
 
 /// A type that is able to track dependencies between tasks.
 package protocol DependencyTracker: Sendable {
-  /// Whether the task described by `self` needs to finish executing before
-  /// `other` can start executing.
-  func isDependency(of other: Self) -> Bool
```

Comment:
Could keep both requirements and add a default implementation of `dependencies(in:)` that does the filtering. Don't really mind though

Reply:
I think it’s good to make adopters a little more aware of the potentially quadratic issue.